pkg/master/thirdparty_controller_test.go

```diff
@@ -90,8 +116,8 @@ func TestSyncAPIs(t *testing.T) {
 				Items: resourcesNamed("foo.example.com"),
 			},
 			apis: []string{
-				"/apis/example.com",
-				"/apis/example.com/v1",
+				"/apis/example.com/foos",
```

Comment:
@lavalamp Agreed. You are right. I take back my previous comment. I will have to fix my tests to make that evident. I rechecked the deployed version of my code and it does not create a webservice without version, which is what we want.

```bash
[uruddarraju@kubernetes-master-4-510999 ~]$ curl http://127.0.0.1:8080/apis/tess.io/clusters
{
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {},
  "status": "Failure",
  "message": "the server could not find the requested resource",
  "reason": "NotFound",
  "details": {},
  "code": 404
}[uruddarraju@kubernetes-master-4-510999 ~]$
[uruddarraju@kubernetes-master-4-510999 ~]$
[uruddarraju@kubernetes-master-4-510999 ~]$ curl http://127.0.0.1:8080/apis/tess.io/v1
{
  "kind": "APIResourceList",
  "apiVersion": "v1",
  "groupVersion": "tess.io/v1",
  "resources": [
    {
      "name": "allocationpools",
      "namespaced": true,
      "kind": "AllocationPool"
    },
    {
      "name": "clusterdeployments",
      "namespaced": true,
      "kind": "ClusterDeployment"
    },
    {
      "name": "clusters",
      "namespaced": true,
      "kind": "Cluster"
    },
    {
      "name": "computenodes",
      "namespaced": true,
      "kind": "ComputeNode"
    },
    {
      "name": "nodepools",
      "namespaced": true,
      "kind": "NodePool"
    }
  ]
}
```